pytorch_lightning/core/lightning.py

```diff
@@ -83,7 +83,7 @@ def forward(self, x):
 
         """
         if self.trainer.proc_rank == 0:
-            print(*args, **kwargs)
+            log.info(*args, **kwargs)
```

Comment:
```suggestion
            log.info(*args, **kwargs)
        else:
            log.debug(*args, **kwargs)
```

Reply:
the point is to only print from proc 0... not any other process